src/scss/skin-modern/components/_subtitleoverlay.scss

```diff
@@ -6,14 +6,22 @@
 
   @include hidden;
 
-  bottom: 2em;
+  align-items: flex-end;
+  display: flex;
   font-size: 1.2em;
-  left: 3em;
+  height: 86%;
+  justify-content: center;
+  margin: auto 10%;
   pointer-events: none;
   position: absolute;
-  right: 3em;
-  text-align: center;
-  transition: bottom $animation-duration-short ease-out;
+  top: 50%;
```

Comment:
As long as there's no label inserted, why does it matter that it doesn't have a height in that case? Did relative positioning not work?